components/ActionSheet/examples/ActionSheetComparisonExampleViewController.m

```diff
@@ -27,61 +30,56 @@ @interface ActionSheetComparisonExampleViewController ()
 
 @end
 
-@implementation ActionSheetComparisonExampleViewController {
-  MDCButtonScheme *_buttonScheme;
-}
+@implementation ActionSheetComparisonExampleViewController
 
 - (instancetype)init {
   self = [super init];
   if (self) {
     self.title = @"Action Sheet";
-    _colorScheme = [[MDCSemanticColorScheme alloc] init];
-    _typographyScheme = [[MDCTypographyScheme alloc] init];
+    _containerScheme = [[MDCContainerScheme alloc] init];
     _showMaterialButton = [[MDCButton alloc] init];
     _showUIKitButton = [[MDCButton alloc] init];
-    _buttonScheme = [[MDCButtonScheme alloc] init];
   }
   return self;
 }
 
-- (MDCContainerScheme *)containerScheme {
-  if (!_containerScheme) {
-    _containerScheme = [[MDCContainerScheme alloc] init];
-  }
-  _containerScheme.colorScheme = self.colorScheme;
-  _containerScheme.typographyScheme = self.typographyScheme;
-  return _containerScheme;
-}
-
 - (void)viewDidLoad {
   [super viewDidLoad];
 
-  self.view.backgroundColor = _colorScheme.backgroundColor;
-  [_showMaterialButton setTitle:@"Show Material Action sheet" forState:UIControlStateNormal];
-  [_showMaterialButton sizeToFit];
-  [_showUIKitButton setTitle:@"Show UIKit Action sheet" forState:UIControlStateNormal];
-  [_showUIKitButton sizeToFit];
-  _buttonScheme.colorScheme = _colorScheme;
-  _buttonScheme.typographyScheme = _typographyScheme;
-  [MDCContainedButtonThemer applyScheme:_buttonScheme toButton:_showMaterialButton];
-  [_showMaterialButton addTarget:self
+  if (self.containerScheme.colorScheme == nil) {
+    self.containerScheme.colorScheme =
+        [[MDCSemanticColorScheme alloc] initWithDefaults:MDCColorSchemeDefaultsMaterial201804];
+  }
+
+  if (self.containerScheme.typographyScheme == nil) {
+    self.containerScheme.typographyScheme =
+        [[MDCTypographyScheme alloc] initWithDefaults:MDCTypographySchemeDefaultsMaterial201804];
+  }
```

Comment:
These lines of code shouldn't be necessary; the theming extensions are expected to handle the nil case.

Reply:
Ah; thanks for clarifying. For future examples, I'd consider theming the view controller's background color as a separate body of work from moving to the container scheme and would avoid adding this code overhead as a result.

Reply:
That isn't being added it was already present in the example. Please see line 59 of the base version:


```objc
self.view.backgroundColor = _colorScheme.backgroundColor;
```

Reply:
I also think it's unnecessary extra work to only migrate *part* of a single example to container schemes and then have to do a second PR for the same file just to migrate the example's color/typography to the container scheme.  I think if we're adding Container Scheme support to an example, we should just move it all over at once.

Reply:
I believe this has been addressed.